shared/src/main/java/uk/gov/di/authentication/shared/services/AccessTokenService.java

```diff
@@ -81,4 +113,39 @@ public Optional<AccessTokenStore> setAccessTokenTtlTestOnly(String accessToken,
                             return ts;
                         });
     }
+
+    @Override
+    public Optional<AccessTokenStore> get(String partition) {
+        Key partitionKey = Key.builder().partitionValue(partition).build();
+        Optional<AccessTokenStore> accessTokenStore =
+                Optional.ofNullable(dynamoTable.getItem(partitionKey));
+        incrementCloudwatchCounter(ACCESS_TOKEN_SERVICE_INITIAL_QUERY_ATTEMPT.getValue());
+
+        if (accessTokenStore.isPresent()) {
+            incrementCloudwatchCounter(ACCESS_TOKEN_SERVICE_INITIAL_QUERY_SUCCESS.getValue());
+        } else {
```

Comment:
No need to change, but just as a general comment I personally find an early return pattern more readable in situations like this